.github/workflows/test_terraform_module.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: Run Terraform tests
+
+on:
+  pull_request:
+    paths:
+      - 'terraform/**'
+
+permissions:
+  contents: read
+
+jobs:
+  test-terraform:
+    name: Test Terraform with Juju
+    runs-on: ubuntu-latest
+    env:
+      WORKING_DIR: 'terraform/tests'
+    steps:
+    - uses: actions/checkout@v4
+    - uses: charmed-kubernetes/actions-operator@main
+      with:
+        provider: "microk8s"
+        channel: 1.28-strict/stable
+        juju-channel: 3.4/stable
```

Comment:
3.6/stable is the LTS

Reply:
The integration tests run in juju-channel: 3.4/stable so I put the same value until we change the integration tests too.